content/src/controller/Controller.ts

```diff
@@ -119,76 +119,78 @@ export class Controller {
     const migrationInformation = JSON.parse(req.body.migration_data)
     const files = req.files
 
-    let deployFiles: ContentFile[] = []
-    try {
-      const auditInfo: LocalDeploymentAuditInfo = {
-        authChain,
-        version: CURRENT_CONTENT_VERSION,
-        migrationData: {
-          originalVersion,
-          data: migrationInformation
-        }
+    const auditInfo: LocalDeploymentAuditInfo = {
+      authChain,
+      version: CURRENT_CONTENT_VERSION,
+      migrationData: {
+        originalVersion,
+        data: migrationInformation
       }
+    }
 
-      if (files instanceof Array) {
-        deployFiles = await Promise.all(files.map((f) => this.readFile(f.fieldname, f.path)))
-      }
-      const creationTimestamp = await this.service.deployLocalLegacy(deployFiles, entityId, auditInfo)
-      res.send({
-        creationTimestamp: creationTimestamp
-      })
-    } catch (error) {
-      Controller.LOGGER.warn(`Returning error '${error.message}'`)
-      res.status(500).send(error.message) // TODO: Improve and return 400 if necessary
-    } finally {
-      await this.deleteUploadedFiles(deployFiles)
+    const deployFiles: ContentFile[] = await this.readFiles(files)
+    const creationTimestamp: Timestamp | ErrorList = await this.service.deployLocalLegacy(
```

Comment:
Also, since we are not sure what is returned, we should probably change the name `creationTimestamp`